CSSS/Kernel.cs

```diff
@@ -166,6 +171,32 @@ public bool PerformTasks()
                 PerformIssueCheckTasks();
             }
 
+            // If we should shutdown
+            if (shouldShutdown)
+            {
+                // Run the shutdown command
+                var process = new System.Diagnostics.Process();
+                process.StartInfo.FileName = "shutdown";
+                switch (config.operatingSystemType)
+                {
+                    case Config.OperatingSystemType.WinNT:
+                        // On Windows, we can specify the seconds
+                        // to delay the shutdown :D
+                        // So we shutdown in 15 seconds (should be suffficient time)
+                        process.StartInfo.Arguments = "-s -t 15";
+                        break;
+                    case Config.OperatingSystemType.Linux:
+                        // Linux doesn't let you specify seconds... >:(
+                        process.StartInfo.Arguments = "-h +1";
+                        break;
+                    default:
+                        logger.Warn("Unknown OS specified in Shutdown - this shouldn't happen");

```

Comment:
Could this be changed to `logger.Warn("Unable to shut down the computer - you will need to do this manually");`?

I have a feeling saying "`this shouldn't happen`" means it somehow will happen (feel free to call me superstitious).